src/resourcetiming-decompression.js

```diff
@@ -782,6 +785,43 @@
         return resource;
     };
 
+  /**
+   * Decompresses arbitrary data back into the specified resource
+   *
+   * @param {string} compressed Compressed string
+   * @param {ResourceTiming} resource ResourceTiming object
+   * @returns {ResourceTiming} ResourceTiming object with decompressed sizes
+   */
+    ResourceTimingDecompression.decompressNamespacedData = function(compressed, resource) {
+        resource = resource || {};
+
+        if (typeof compressed === "string") {
+            var delimiter = ":";
+            var colon = compressed.indexOf(delimiter);
+            if (colon > 0) {
```

Comment:
Do we want to allow for `name` pairs in addition to `name:value`?

If `value` is missing, it's assumed to be `1` or `true` or something?

Reply:
We could. Complicates the logic and documentation a bit. But does make more consumable.

Reply:
Let's not do it now then, we can always add it later